src/data/routing/Routing.js

```diff
@@ -0,0 +1,46 @@
+/*
+
+@params
+Input
+data: Object
+
+Output
+percentage: string
+
+@description
+Calculate the usage % of driver.
+
+*/
+
+export var calculateDriverUsage = function(data) {
+
+    if (!data) {
+        throw 'data is empty';
```

Comment:
- Please use `new` keyword when throwing an error (https://google.github.io/styleguide/jsguide.html#features-exceptions)
- You should use a `try...catch` block whenever you use the `throw` keyword